end-to-end-tests/src/bin/bootstrap.rs

```diff
@@ -41,12 +41,14 @@ async fn main() -> Result<()> {
     eprintln!("ensuring datasets are ready...");
     let ctx = Context::from_client(client).await?;
     let disk_name = generate_name("disk")?;
+    let org_name = NameOrId::Name(ctx.org_name.clone());
+    let project_name = NameOrId::Name(ctx.project_name.clone());
```

Comment:
This is all kind of verbose. The async block seems to mess with its ability to infer a simple `.into()`.

<img width="918" alt="Screenshot 2023-03-17 at 2 53 55 PM" src="https://user-images.githubusercontent.com/3612203/226018527-8898c18f-f4aa-427b-90a3-53ce423138a7.png">

Reply:
Well this solves the entire problem.

https://github.com/oxidecomputer/typify/pull/224